.vscode/settings.json

```
@@ -0,0 +1,30 @@
+{
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "quickfix.biome": "explicit",
```

quickfix 는 처음 보는 설정인데
```suggestion
    "source.fixAll.biome": "explicit",
```

와 어떻게 다른가요? 🤔

---

둘 다 vscode에서 함께 제공하는 속성인데 속성명처럼 quickfix가 더 세부적인 부분을 수정하는 것 같아요 (quickfix : `command + .` 기능)
biome에서는 save on action을 quickfix로 설정하더라구요.!